ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Raw.scala

```diff
@@ -73,4 +73,6 @@ object Raw {
   final case class ContractInstance(override val bytes: ByteString) extends Bytes
 
   object ContractInstance extends Companion[ContractInstance]
+
+  final case class NodeId(value: String) extends AnyVal
```

Comment:
are you worried about the costs of converting the int to a string or why can’t we rely on the existing `NodeId` type?

Reply:
I am guessing that if we just use the `NodeId` type directly, it will break processing historical data.

Reply:
that’s what I was asking about conversions. If you serialize at an int you clearly break stuff but if you convert the string to an int at the border it seems like it should be fine?

Reply:
either way maybe worth documenting the reason in a comment

Reply:
The reason is `DamlTransactionBlindingInfo.DisclosureEntry` has `string node_id = 1;`. I could change it to a dedicated message with an int, as we ignore type changes in our buf checks (for now). However, it would still need a conversion, so I would go with the current change. Not sure if it's worth commenting. WDYT?

Reply:
I’d add the comment just because I know that I would be asking myself in the future why there is a separate type but happy to leave this up to your team.